package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "hof",
   "description": "A bootstrap for HOF projects",
-  "version": "20.4.5",
+  "version": "20.4.6-add-checkbox-hint-beta",
```

Comment:
you won't be publishing this as a beta version so you don't need to name it as such. I would make it a minor version i.e. 20.5.0. as you are adding enhancements.

Reply:
Renamed the version to 20.5.0